src/frontends/onnx/README.md

```diff
@@ -0,0 +1,66 @@
+# OpenVINO ONNX Frontend
+
+The main responsibility of the ONNX Frontend is to import ONNX models and convert them into the `ov::Model` representation. 
+Other capabilities of the ONNX Frontend:
+* modification of tensors properties (like data type and shapes)
+* changing topology of the models (like cutting subgraphs, inserting additional inputs and outputs)
+* searching the models in a user-friendly way via tensors and operators names
+
+The component is written in `C++`. `Python` bindings are also available.
```

Comment:
What do you mean: "`Python` bindings are also available."? If I remember right we don't have ONNX Frontend specific objects in python (excepts the extension)

Reply:
I mean here common FE API bindigs here, but you are right, that it can be confused. Removed.